src/style.css

```diff
@@ -1 +1,83 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
```

Comment:
This setting is better suited exactly where needed - in the link styles. You don't want to set it for the whole project.